packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,245 @@
+
+## Anatomy
+
+<img src="./bottomsheet-anatomy.png" alt="BottomSheet Anatomy" width="100%" />
+
+
+Components:
+
+- BottomSheet
+  - BottomSheetHeader
+  - BottomSheetBody
+  - BottomSheetFooter
+
+## API
+
+Sample usage:
+
+```jsx
+import { BottomSheet } from '@razorpay/blade';
+
+<BottomSheet isOpen={boolean} snapPoints={[]}>
+  <BottomSheetHeader
+    title="Select Account"
+    leading={<BladeIcon />}
+  />
+  <BottomSheetBody>
+    <Text>Body Content<Text>
+  </BottomSheetBody>
+  <BottomSheetFooter
+    title="Footer Title"
+    leading={BladeIcon}
+    trailing={
+      <>
+        BladeIcon | IconButton | Button
+      </>
+    }
+  />
+</BottomSheet>;
+```
+
+**Design Constraints**
+
+- BottomSheetHeader:
+  - title
+  - leading:
+    - BladeIcon
+
+- BottomSheetFooter:
+  - title
+  - leading:
+    - BladeIcon
+  - trailing
+    - any jsx element
+
+- The BottomSheetFooter will behave similar to ActionListFooter.
+- The Header, Footer can be individually omitted
+
+### `BottomSheet`
+
+We'll expose a `BottomSheet` component with the following API:
+
+| Prop            | Type        | Default                 | Description                                                                                     | Required |
+| --------------- | ----------- | ----------------------- | ----------------------------------------------------------------------------------------------- | -------- |
+| isOpen            | `boolean`   | `false`                 | Toggles bottom sheet state                                                              |          |
+| snapPoints      | `[number, number, number]`  | `[0.35, 0.5, 0.85]` | Snappoints in which the bottom sheeet will rest on, this accepts a number between 0 & 1 which maps to the total view height of the screen. 0.5 means 50% of screen height.                                              |          |
+| onDismiss       | `Callback`  | `undefined`             | called when the bottom sheet is closed, either by user state, hitting `esc` or tapping backdrop |          |
+| initialFocusRef | `React.Ref` | `undefined`             | ref element you want to get keyboard focus when opening the sheet                               |          |
+
+### `BottomSheetBody`
+
+| Prop       | Type              | Default     | Description                 | Required |
+| ---------- | ----------------- | ----------- | --------------------------- | -------- |
+| `children` | `React.ReactNode` | `undefined` | Contents of the BottomSheet | ✅       |
+
+### `BottomSheetHeader` API
+
+| Prop     | Type            | Default     | Description                          | Required |
+| -------- | --------------- | ----------- | ------------------------------------ | -------- |
+| `title`  | `string`        | `undefined` | Title of the Header                    | ✅       |
+| `leading` | `IconComponent` | `undefined` | leading icon placed before title text |          |
```

Comment:
leading icon is enough? we don't have a use case for badge or anything else? did we check with Merchant Dashboard and SME team? Can you check with Abhishek?